ci/requirements-3.6.run

```diff
@@ -14,7 +14,7 @@ html5lib
 jinja2
 sqlalchemy
 pymysql
-feather-format
+# feather-format (not available on defaults ATM)
```

Comment:
no, the 3.6 run specially uses conda-forge (this is the only run that does this)